code/modules/economy/credits.dm

```diff
@@ -48,6 +48,24 @@
 	.=..()
 	set_worth(round(rand_between(initial(worth)*0.5, initial(worth)*1.5), 1))
 
+/obj/item/weapon/spacecash/ewallet/random/c1
+	worth = 1
```

Comment:
This is awful for gameplay. Imagine finding A SINGLE PENNY. Oh boy, only need 3999 more of those to actually buy a linerack. This is something that could spawn in a random trash spawner, but not a cash spawner. With how expensive things are in the store, there's a reason the minimum is 200

Reply:
That reasoning was never really elaborated on to me. So I offered this economy overhaul. 200 minimum is still quite a lot.